src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1098,6 +1098,14 @@
     <value>If checked, show all installed fonts in the list above. Otherwise, only show the list of monospace fonts.</value>
     <comment>A description for what the supplementary "show all fonts" setting does. Presented near "Profile_FontFaceShowAllFonts".</comment>
   </data>
+  <data name="Profile_CreateUnfocusedAppearanceButton.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+    <value>Create an unfocused appearance for this profile.</value>

```

Comment:
Can you further describe what an unfocused appearance is? Would that help folks to know without having to go Bing/Google it for our docs? "This is a style applied to the window when it loses primary focus." or something?

Reply:
Doesn't Windows describe it as Active and Inactive windows?

Reply:
Added a little more description!